### PR TITLE
Initial draft of OIE concept doc

### PR DESCRIPTION
**Pull request by RobertAnderson-okta**
## Description:
- **What's changed?** Initial draft of OIE concept doc
- **Is this PR related to a Monolith release?** no

### Resolves:

* [OKTA-358659](https://oktainc.atlassian.net/browse/OKTA-358659)

### CODE REVIEW

**Comment by jakubvul-okta** [comment]
Looking at this, there seems to be an opportunity to include 2-3 scenarios (probably from the user's perspective) that are enabled by OIE. Things that you can do now that you weren't able to do, from the CIAM world.

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Identity engine allows organizations to customize their Okta cloud components to satisfy an unlimited number of identity use cases. Instead of a reliance on pre-defined behavior for authentication, authorization, and enrollment, Identity engine offers customizable building blocks that can support dynamic, app-based user journeys.
+
+Identity engine provides greater fine-grained control when compared to traditional access flows. It enforces custom requirements at every point in the end user's sign-in experience &mdash; from registration and authentication to enrollment and authorization.
+
+Using the Admin Console, you can direct the user experience based on user type, group membership, and target application. Since Identity engine can execute Okta Hooks or redirect to external services, you can also extend that experience further within the security guardrails of Okta.
+
+## Why use Identity engine?
+
+Identity engine has been built around three major tenets:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:20]:
```suggestion
Identity engine is built around three major tenets:
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+* Passwordless authentication
+
+  Admins can enable the email authenticator now with the addition of a magic link in the email notification sent to end users. When policies are configured to include non-password authenticators, end users may sign in to their account using factors that do not require use of a password. For more information, see [Configure passwordless authentication with email magic link](https://help.okta.com/en/oie/Content/Topics/identity-engine/procedures/configure-passwordless-auth.htm).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:24]:
```suggestion
  Admins can enable the email authenticator now with the addition of a magic link in the email notification sent to end users. When policies are configured to include non-password authenticators, end users may sign in to their account using factors that don't require the use of a password. See [Configure passwordless authentication with email magic link](https://help.okta.com/en/oie/Content/Topics/identity-engine/procedures/configure-passwordless-auth.htm).
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+* Progressive Profiling
+
+  Update an existing user's profile by prompting them for additional sign-in information when they advance to designated points. For more information, see [Create a Profile Enrollment policy for progressive profiling](https://help.okta.com/en/oie/Content/Topics/identity-engine/policies/create-profile-enrollment-policy-pp.htm).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:28]:
```suggestion
  Update an existing user's profile by prompting them for additional sign-in information when they advance to designated points. See [Create a Profile Enrollment policy for progressive profiling](https://help.okta.com/en/oie/Content/Topics/identity-engine/policies/create-profile-enrollment-policy-pp.htm).
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+## Why use Identity engine?
+
+Identity engine has been built around three major tenets:
+
+* Passwordless authentication
+
+  Admins can enable the email authenticator now with the addition of a magic link in the email notification sent to end users. When policies are configured to include non-password authenticators, end users may sign in to their account using factors that do not require use of a password. For more information, see [Configure passwordless authentication with email magic link](https://help.okta.com/en/oie/Content/Topics/identity-engine/procedures/configure-passwordless-auth.htm).
+
+* Progressive Profiling
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:26]:
```suggestion
* Progressive profiling
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+  Update an existing user's profile by prompting them for additional sign-in information when they advance to designated points. For more information, see [Create a Profile Enrollment policy for progressive profiling](https://help.okta.com/en/oie/Content/Topics/identity-engine/policies/create-profile-enrollment-policy-pp.htm).
+
+* App-level policies
+
+  App sign-on policies define the full requirements for an app. Admins can configure Okta Sign-on policies to use App-level policies instead &mdash; making it easier to manage your apps. For more information, see [App sign-on policies](https://help.okta.com/en/oie/Content/Topics/identity-engine/policies/about-app-sign-on-policies.htm).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:32]:
```suggestion
  App sign-on policies define the full requirements for an app. Admins can configure Okta Sign-On Policies to use App-level policies instead &mdash; making it easier to manage your apps. See [App sign-on policies](https://help.okta.com/en/oie/Content/Topics/identity-engine/policies/about-app-sign-on-policies.htm).
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+## New concepts and terminology
+
+Identity engine introduces new concepts and terminology.
+
+* Assurance
+
+  Assurance is the degree of confidence that an end user signing in to an application is the same end user who previously signed in to the application, and the use of one or more authenticators (see below) and its characteristics determine an assurance level. For example, an end user who authenticates with a knowledge factor and a possession factor has a higher assurance level than one who can only authenticate with one factor.
+
+  Assurance is enforced by sign-on policies. OIE requires that the assurance specified in the Okta and app sign-on policies are satisfied before it allows the end user to access an app. This is a change from the traditional model of authentication, which evaluates one policy depending on whether the user signs in to the org or directly through the app.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:42]:
```suggestion
  Assurance is enforced by sign-on policies. Identity engine requires that the assurance specified in the Okta and app sign-on policies are satisfied before it allows the end user to access an app. This is a change from the traditional model of authentication, which evaluates one policy depending on whether the user signs in to the org or directly through the app.
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+* Authenticators
+
+  Authenticators are credentials owned or controlled by an end user which can be verified by an application or service. Passwords, answers to security questions, phones (SMS or voice), and authentication apps like Okta Verify are examples of authenticators.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:46]:
```suggestion
  Authenticators are credentials owned or controlled by an end user that can be verified by an application or service. Passwords, answers to security questions, phones (SMS or voice), and authentication apps like Okta Verify are examples of authenticators.
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+* Factor Types
+
+  Okta is changing its definition of authenticators and factors in order to provide an industry-standard differentiation:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:50]:
```suggestion
  Okta is changing its definition of authenticators and factors to provide an industry-standard differentiation:
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+
+  * Factors are different categories that define how authentication takes place and the means in which they are controlled by end users.
+  * Authenticators are used to verify one or more multiple factors such as Knowledge (for example, passwords and security questions), Possession (for example, Email, SMS, Okta Verify, and hardware tokens), and Inherence/Biometrics (for example, fingerprints and facial recognition).
+  * A sign-on policy with two-factor authentication requires two distinct factors. Two authenticators of the same factor (Knowledge, Possession, or Inherence) will not be accepted.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:54]:
```suggestion
  * A sign-on policy with two-factor authentication requires two distinct factors. Two authenticators of the same factor (Knowledge, Possession, or Inherence) isn't accepted.
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+
+* Okta sign-on policies
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:56]:
```suggestion
* Okta Sign-On Policies
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+
+  Assurance is the degree of confidence that an end user signing in to an application is the same end user who previously signed in to the application, and the use of one or more authenticators (see below) and its characteristics determine an assurance level. For example, an end user who authenticates with a knowledge factor and a possession factor has a higher assurance level than one who can only authenticate with one factor.
+
+  Assurance is enforced by sign-on policies. OIE requires that the assurance specified in the Okta and app sign-on policies are satisfied before it allows the end user to access an app. This is a change from the traditional model of authentication, which evaluates one policy depending on whether the user signs in to the org or directly through the app.
+
+* Authenticators
+
+  Authenticators are credentials owned or controlled by an end user which can be verified by an application or service. Passwords, answers to security questions, phones (SMS or voice), and authentication apps like Okta Verify are examples of authenticators.
+
+* Factor Types
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:48]:
```suggestion
* Factor types
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+
+  Okta is changing its definition of authenticators and factors in order to provide an industry-standard differentiation:
+
+  * Factors are different categories that define how authentication takes place and the means in which they are controlled by end users.
+  * Authenticators are used to verify one or more multiple factors such as Knowledge (for example, passwords and security questions), Possession (for example, Email, SMS, Okta Verify, and hardware tokens), and Inherence/Biometrics (for example, fingerprints and facial recognition).
+  * A sign-on policy with two-factor authentication requires two distinct factors. Two authenticators of the same factor (Knowledge, Possession, or Inherence) will not be accepted.
+
+* Okta sign-on policies
+
+  Okta sign-on policies are globally applied to all applications in the tenant, and specify actions to take, such as allowing access, prompting for a challenge, setting the time before prompting for another challenge, and expiring a session. For more information, see [Okta sign-on policies](https://help.okta.com/en/oie/Content/Topics/identity-engine/policies/about-okta-sign-on-policies.htm).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:58]:
```suggestion
  Okta Sign-On Policies are globally applied to all applications in the tenant, and specify actions to take, such as allowing access, prompting for a challenge, setting the time before prompting for another challenge, and expiring a session. See [Okta Sign-On Policies](https://help.okta.com/en/oie/Content/Topics/identity-engine/policies/about-okta-sign-on-policies.htm).
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+
+* App sign-on policies
+
+  App sign-on policies enforce authentication when an end user attempts to access an application. Each application is assigned one app sign-on policy (with multiple rules that may be ordered). For more information, see [App sign-on policies](https://help.okta.com/en/oie/Content/Topics/identity-engine/policies/about-app-sign-on-policies.htm).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:62]:
```suggestion
  App sign-on policies enforce authentication when an end user attempts to access an application. Each application is assigned one app sign-on policy (with multiple rules that you can order). See [App sign-on policies](https://help.okta.com/en/oie/Content/Topics/identity-engine/policies/about-app-sign-on-policies.htm).
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+
+## Enable Identity engine for your organization
+
+To upgrade to Identity engine, please reach out to your account manager. If you do not have an account manager, please reach out to oie@okta.com for more information.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:66]:
```suggestion
To upgrade to Identity engine, reach out to your account manager. If you don't have an account manager, reach out to oie@okta.com for more information.
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+
+* The v1 API will continue to work as before until you're ready to use new Identity engine functionality.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:68]:
```suggestion
* The v1 API continues to work as before until you're ready to use new Identity engine functionality.
```

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+* The existing Okta-hosted widget will continue to work after upgrading your org.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:69]:
```suggestion
* The existing Okta-hosted widget continues to work after upgrading your org.
```

**Review by susanharper-okta — COMMENTED** [review]
couple style things

File: packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md [hunk 17]
```diff
@@ -31,6 +31,10 @@ Identity engine is built around three major tenets:
 
   App sign-on policies define the full requirements for an app. Admins can configure Okta Sign-On Policies to use App-level policies instead &mdash; making it easier to manage your apps. See [App sign-on policies](https://help.okta.com/en/oie/Content/Topics/identity-engine/policies/about-app-sign-on-policies.htm).
 
+* MFA enrollment policies
+
+  With MFA enrollment policies, you can create and enforce policies and rules for specific MFA factors and assign groups accordingly. Sign-on policies determine the types of authentication challenges end users experience when they sign into their account. MFA enrollment policies are based on a variety of factors, such as location, group definitions, and authentication type. See [Create an MFA enrollment policy]([ddd](https://help.okta.com/en/oie/Content/Topics/identity-engine/policies/create-mfa-policy.htm)).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md:36]: they sign into -> they sign in to

Sign-on policies determine the types of authentication challenges end users experience when they sign in to their account.

**Review by susanharper-okta — COMMENTED** [review]
just one thing

File: packages/@okta/vuepress-theme-prose/const/navbar.const.js [hunk 18]
```diff
@@ -38,6 +38,11 @@ export const concepts = [
         title: "How Okta works",
         path: "/docs/concepts/how-okta-works/"
       },
+      {
+        title: "Identity engine",
+        path: "/docs/concepts/ie-intro/",
+        hidden: true
```
Comment (albertchen-okta) [inline packages/@okta/vuepress-theme-prose/const/navbar.const.js:44]: are we hiding this concept @jakubvul-okta @emekaafigbo-okta ? One big reason for the concept is so we can link OIE guides and references to this concept. Does it make sense to keep it hidden from the nav?

**Pull request merged**